br/pkg/lightning/backend/external/writer.go

```diff
@@ -47,7 +47,7 @@ var (
 
 	// MergeSortOverlapThreshold is the threshold of overlap between sorted kv files.
 	// if the overlap ratio is greater than this threshold, we will merge the files.
-	MergeSortOverlapThreshold int64 = 1000
+	MergeSortOverlapThreshold int64 = 8000
```

Comment:
for import, write&ingest step will adjust concurrency to read ingest-data depends on len(dataFiles). too many files might limit write speed.
maybe use another var on add-index
https://github.com/pingcap/tidb/blob/07fd1a433fd355880891779b7ec2a6e25f6829a0/br/pkg/lightning/backend/external/engine.go#L151

Reply:
5b629ebb65e3d7af26eeeb750fbb33f535112e1e